zone/bot_command.cpp

```diff
@@ -10037,7 +10041,7 @@ void bot_command_pickpocket(Client *c, const Seperator *sep)
 	glm::vec4 mob_distance    = (c->GetPosition() - target_mob->GetPosition());
 	float     mob_xy_distance = ((mob_distance.x * mob_distance.x) + (mob_distance.y * mob_distance.y));
 	float     mob_z_distance  = (mob_distance.z * mob_distance.z);
-	if (mob_z_distance >= 25 || mob_xy_distance > 250) {
+	if (mob_z_distance >= 40 || mob_xy_distance > 250) {
```

Comment:
This should use `GetZOffset` it takes in relation the model and the size of the mob - otherwise you will always find yourself tweaking this number

Reply:
cc @tuday2

Reply:
Thanks did not know about this, I changed the code and tested. We should be good.